### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception Cannot read scheme com.intellij.configurationStore.SchemeManagerIprProvider: AZURE_FUNCTION_SUPPORT-Deploy Functions_ ues-creditrisk-service_creditrisk-function-app_ factoryName_ Deploy Functions.xml com.intellij.util.xmlb.XmlSerializationException: Cannot deserialize class com.microsoft.azure.toolkit.intellij.legacy.function.runner.deploy.FunctionDeployModel

### ISSUE THREAD

**Issue by rtvoelker** [microsoft/azure-tools-for-java#6670]
IntelliJ build version: 2022.1.1 IU-221.5591.52
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.14.1
Plugin version: 3.65.0-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1512x944,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.ui.ToolbarUtil$3@660fc22c,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=Cannot read scheme com.intellij.configurationStore.SchemeManagerIprProvider: AZURE_FUNCTION_SUPPORT-Deploy Functions_ ues-creditrisk-service_creditrisk-function-app_ factoryName_ Deploy Functions.xml, throwable=com.intellij.util.xmlb.XmlSerializationException: Cannot deserialize class com.microsoft.azure.toolkit.intellij.legacy.function.runner.deploy.FunctionDeployModel
	at com.intellij.util.xmlb.XmlSerializer.deserialize(XmlSerializer.java:58)
	at com.microsoft.azure.toolkit.intellij.legacy.function.runner.deploy.FunctionDeployConfiguration.lambda$readExternal$0(FunctionDeployConfiguration.java:170)
	at com.microsoft.azure.toolkit.intellij.legacy.function.r

**Comment by wangmingliang-ms** [comment]
this exception should be fixed by commit 35d3dbd2 and the fix will be included in the next stable release.

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
